angular-frontend/src/app/omnai-datasource/omnai-scope-server/live-data.service.ts

```diff
@@ -25,12 +27,14 @@ interface DeviceOverview {
 @Injectable({
   providedIn: 'root'
 })
-export class OmnAIScopeDataService implements DataSource{
+export class OmnAIScopeDataService implements DataSource {
 
   private serverUrl = '127.0.0.1:8080';
+  private readonly destroy$ = new Subject<void>();
```

Comment:
https://angular.dev/guide/components/lifecycle#destroyref

```suggestion
  private readonly destroyRef = inject(DestroyRef);
```